front-end/studio/src/app/pages/apis/{apiId}/editor/_components/dialogs/add-response.component.ts

```diff
@@ -18,7 +18,134 @@
 import {Component, EventEmitter, Output, QueryList, ViewChildren} from "@angular/core";
 import {ModalDirective} from "ngx-bootstrap";
 import {Oas20Operation, Oas30Operation} from "oai-ts-core";
+import {DropDownOption, DropDownOptionValue as Value, DIVIDER} from "../../../../../../components/common/drop-down.component";
 
+export class HttpCode {
+
+    private code: number;
+    private name: string;
+
+    public constructor(code: number, name: string) {
+        this.code = code;
+        this.name = name;
+    }
+
+    public getCode(): number {
+        return this.code;
+    }
+
+    public getName(): string {
+        return this.name;
+    }
+}
+
+export class HttpCodeUtil {
```

Comment:
Can you leverage this here?

https://github.com/Apicurio/apicurio-studio/blob/master/front-end/studio/src/app/pages/apis/%7BapiId%7D/editor/_services/httpcode.service.ts